music21/stream/tests.py

```diff
@@ -2220,8 +2220,8 @@ def testReplace(self):
         s.replace(n4, n1)
         self.assertEqual([s[0], s[1]], [n3, n1])
 
-        error_msg = f'{n3} already in {s}'
-        with self.assertRaises(StreamException, msg=error_msg):
+        expected = f'{n3} already in {s}'
+        with self.assertRaisesRegex(StreamException, expected):
```

Comment:
Thank you for these assert changes -- I had no idea that they weren't working!

Reply:
yeah, it was my bad...